python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1906,6 +1941,23 @@ router bgp {{ router_bgp.as }}
 {%                 if neighbor.route_map_out is arista.avd.defined %}
          neighbor {{ neighbor.ip_address }} route-map {{ neighbor.route_map_out }} out
 {%                 endif %}
+{%                 if neighbor.additional_paths.receive is arista.avd.defined %}
+         neighbor {{ neighbor.ip_address }} additional-paths receive
+{%                 endif %}
+{%                 if neighbor.additional_paths.send is arista.avd.defined %}
```

Comment:
can you check again as i dont see the schema for send in vrf.address_family_ipv4_multicast